data_registry/process_manager/task/coverage.py

```diff
@@ -0,0 +1,64 @@
+import gzip
+import shutil
+import tempfile
+from pathlib import Path
+
+import ocdscardinal
+
+from data_registry import models
+from data_registry.process_manager.util import TaskManager, skip_if_not_started
+from exporter.util import Export
+
+
+def get_keys_for_subschema(coverage, subschema):
+    return (key for key in coverage if key.endswith(subschema))
+
+
+class Coverage(TaskManager):
+    final_output = True
+
+    def get_export(self):
+        return Export(self.job.pk, "coverage")
+
+    def run(self):
```

Comment:
I just remembered that we need to put all of this in a worker.

The TaskManager is run by the manageprocess command, which is designed to run every 5 minutes and return very quickly. It instead just publishes messages for a coverage worker (exporter/management/commands) to read.

Reply:
Thanks, this was the type of feedback I was looking for :) will move this under commands